docs/self-managed/zeebe-gateway-deployment/the-zeebe-gateway.md

```diff
@@ -0,0 +1,169 @@
+|-----------------------------------------------------|-----------------------------------------------------|------------------------------------------------------------------------------------------------------|---------------|
+| `ZEEBE_GATEWAY_CLUSTER_SECURITY_ENABLED`              | `zeebe.gateway.cluster.security.enabled`              |Enables TLS authentication between this gateway and other nodes in the cluster.| `false`         |
+| `ZEEBE_GATEWAY_CLUSTER_SECURITY_CERTIFICATECHAINPATH` | `zeebe.gateway.cluster.security.certificateChainPath` |Sets the path to the certificate chain file.|               |
+| `ZEEBE_GATEWAY_CLUSTER_SECURITY_PRIVATEKEYPATH`       | `zeebe.gateway.cluster.security.privateKeyPath`       |Sets the path to the private key file location.|               |
+
+### Message compression
+
+To configure the compression algorithm for all messages sent between the gateway and
+the brokers, the following property can be set. Available options are NONE, GZIP, and SNAPPY.
+This feature is useful when the network latency between the nodes is very high (for example, when nodes are deployed in different data centers). When latency is high, the network bandwidth is severely reduced. Therefore, enabling compression helps improve the throughput.
+
+:::note
+When there is no latency enabling, this may have a performance impact. Additionally, when this flag is enabled, you must also enable compression in the standalone broker configuration.
+:::
+
+| Environment variable                     | Application.yaml property                           | Description                                                                                          | Default value |
+|------------------------------------------|-----------------------------------------------------|------------------------------------------------------------------------------------------------------|---------------|
+| `ZEEBE_GATEWAY_CLUSTER_MESSAGECOMPRESSION` | `zeebe.gateway.cluster.messageCompression` | Configure compression algorithm for all messages sent between the brokers and between the broker and the gateway. Available options are NONE, GZIP, and SNAPPY. | `NONE` |
+
+### Threads configuration
+
+To handle many concurrent incoming requests, the user can do two things: scale the deployed gateways (if the standalone mode is in use), or increase the used resources and threads.
+
+The Zeebe Gateway uses one thread by default, but this should be set to a higher number if the gateway doesn’t exhaust its available resources and doesn’t keep up with the load. The corresponding environment variables look like this: `ZEEBE_GATEWAY_THREADS_MANAGEMENTTHREADS`.
+During benchmarking and when increasing the thread count, it might make sense to also increase the given resources, which are quite small in the Helm chart.
```

Comment:
"Small" in the Helm chart as in don't take up too much space?

Reply:
Maybe low or little is the better term.